src/lang/en-US.ts

```diff
@@ -462,16 +468,28 @@ const AdminPage: ResolvedIntlConfig['messages'] = {
     'admin.billing.paymentMethods.table.label.actions': `Actions`,
     'admin.billing.paymentMethods.table.emptyTableDefault.message': `No payment methods available.`,
 
-    'admin.users.cta.sharePrefix': `Manage Invitations`,
-    'admin.users.sharePrefix.header': `Manage Invitations`,
-    'admin.users.sharePrefix.message': `This is a placeholder for a description.`,
-    'admin.users.sharePrefix.label.capability': `Capability`,
-    'admin.users.sharePrefix.label.type': `Type`,
-    'admin.users.sharePrefix.cta.generateLink': `Generate Invitation`,
+    'admin.users.cta.prefixInvitation': `Manage Invitations`,
+    'admin.users.prefixInvitation.header': `Manage Invitations`,
+    'admin.users.prefixInvitation.message': `This is a placeholder for a description.`,
+    'admin.users.prefixInvitation.label.prefix': `Prefix`,
```

Comment:
We already have two common messages for "Prefix" please use one of those.

Reply:
Oversight. I could not remember what message ID I used before the minor refactoring of `GenerateInvitation`. Pretty sure it was `'common.tenant'`. Will update.